include/build_config/SDL_build_config.h.cmake

```diff
@@ -518,6 +518,9 @@
 #cmakedefine SDL_VIDEO_VITA_PVR @SDL_VIDEO_VITA_PVR@
 #cmakedefine SDL_VIDEO_VITA_PVR_OGL @SDL_VIDEO_VITA_PVR_OGL@
 
+#cmakedefine SDL_SPINLOCK_IS_PTHREAD_MUTEX @SDL_SPINLOCK_IS_PTHREAD_MUTEX@
```

Comment:
If this is to be accepted, `SDL_SPINLOCK_IS_PTHREAD_MUTEX` needs to be defined as 1 in certain standalone configs such as `SDL_build_config_android.h`, `SDL_build_config_macos.h`, and _maybe_ others.

Reply:
As 1 or as 0? I wouldn't want to disturb the implementation on any existing platform unless somebody asks for that.

Reply:
And it looks like the code built successfully for Android and MacOs.

Reply:
Of course it does. And it used to build successfully before this patch too. The thing is: You are _already_ changing existing builds when they are built using cmake, and without updating pthread-using platforms' standalone config files, builds using cmake and builds using project-files or makefiles will output differently, which is a big no.

Reply:
And... I seem to have misread the patchset a bit? You aren't actually defining `SDL_SPINLOCK_IS_PTHREAD_MUTEX` anywhere, so android and macos aren't affected as they currently are. Hm, sorry for the noise.

Where do you actually plan to define `SDL_SPINLOCK_IS_PTHREAD_MUTEX` really though??

Reply:
It's defined for QNX in `CMakeLists.txt`. See pull request #7018